shanoir-ng-preclinical/src/main/java/org/shanoir/ng/preclinical/subjects/dto/PreclinicalSubjectDto.java

```diff
@@ -0,0 +1,43 @@
+package org.shanoir.ng.preclinical.subjects.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(
+        ignoreUnknown = true
+)
+public class PreclinicalSubjectDto {
```

Comment:
I don't get what why this class, some kind of  association class between AnimalSubjectDTO and SubjectDTO ?

Reply:
As discussed with you, this is it : a DTO to represent the association between an animal subject from preclinical and a subject from studies